src/docs/build-and-earn/testnet-guides/create-validator.md

```diff
@@ -35,11 +37,11 @@ You have to wait ~12 hours to finish syncing the Rinkeby test network. Its size
 
 To locate the containers' ID, that is needed to connect to its bash, run the following command in the terminal:
```

Comment:
a couple of things could be improved in this section below:
- if should be clear that it's about the full (validator) node - not the ethereum node from the previous step ?!
- ***from the full node***, the container can be accessed directly by its name - instead of looking for container id then attaching to it, so it can be replaced with eg:

> sudo docker exec -it cudos-start-full-node-client-testnet-public-01 bash

- (example!) `export STAKE="1acudos"` would fail (and users might just copy and paste it) as it has to be ***at least*** `1 000 000 000 000 000 000 acudos`, so it's better replaced with eg:

> export STAKE="1000000000000000000acudos"